.github/workflows/azure-dev.yml

```diff
@@ -52,9 +52,9 @@ jobs:
                   cd ../../client
                   npm link adaptive-templating-service-typescript-node
                   npm install
-                  cd src
-                  npm link adaptive-templating-service-typescript-node
-                  cd ..
+                  # cd src
```

Comment:
did you mean to comment this?

Reply:
we were just trying something with the pipeline, i changed it back